repentogon/resources/scripts/main_ex.lua

```diff
@@ -1635,4 +1692,4 @@ pcall(require("repentogon_extras/daily_stats"))
 pcall(require("repentogon_extras/stats_menu"))
 pcall(require("repentogon_extras/bestiary_menu"))
 -- pcall(require("repentogon_extras/onlinestub")) let's not load it
-pcall(require("repentogon_extras/mods_menu_tweaks"))
\ No newline at end of file
+pcall(require("repentogon_extras/mods_menu_tweaks"))
```

Comment:
I'm not actually sure if this is fixed or how to fix it? I was just putting test callbacks in this file to not have to make a separate mod and since I'd be changing `main_ex.lua` pretty often.